src/blockchain/web3_extentions/requests_metric_middleware.py

```diff
@@ -50,4 +51,15 @@ def middleware(method: RPCEndpoint, params: Any) -> RPCResponse:
 
         return middleware
 
+    web3.middleware_onion.inject(
```

Comment:
add_requests_metric_middleware should not add any cache middlewares

Reply:
Moved out of here